examples/cpp/comm/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ add_subdirectory (event)
 add_subdirectory (generic)
 add_subdirectory (layer)
 add_subdirectory (logging)
+add_subdirectory (multi-trans-units)
```

Comment:
Let me just check in the fix itself for now then. We will need to find a good approach for making sure this kind of thing doesn't regress.